DataFormats/Common/interface/OwnVector.h

```diff
@@ -65,6 +65,7 @@ namespace edm {
       const_iterator & operator +=(difference_type d) { i += d; return *this; }
       const_iterator & operator -=(difference_type d) { i -= d; return *this; }
       reference operator[](difference_type d) const { return *const_iterator(i+d); } // for boost::iterator_range []
+      typename base::const_iterator base_iter() const { return i; }
```

Comment:
I had left it public since the constructor taking a `base::const_iterator` was public.

Should I make both private?